core/domain/email_manager.py

```diff
@@ -203,10 +204,10 @@ def log_new_error(*args, **kwargs):
     'I\'m writing to inform you that I have unpublished the above '
     'exploration.')
 
-NOTIFICATION_EMAILS_FOR_FAILED_TASKS = config_domain.ConfigProperty(
-    'notification_emails_for_failed_tasks',
-    NOTIFICATION_EMAIL_LIST_SCHEMA,
-    'Email(s) to notify if an ML training task fails',
+NOTIFICATION_USER_IDS_FOR_FAILED_TASKS = config_domain.ConfigProperty(
+    'notification_user_ids_for_failed_tasks',
```

Comment:
Has this been populated in production? I don't think we can just change config properties--we'll need to migrate the data.

Reply:
In production (test server or backup server too) this config is not set yet.